### DISCUSSION TITLE
How to add local dependency

### DISCUSSION THREAD

**Discussion by HugoMVale** [fortran-lang/fpm#736] [General]
Hello,
Sorry for the trivial question, whose answer is probably just in front of my nose...
Imagine that I have a _local_ project A stored in folder `./A`, with some modules and so forth. I compile it the usual way with `fpm build` and I get the expected ` .a` and `.mod` files.
Then I have a project B  stored in folder `./B`, where I want to use some modules from project A. What do I need to write in the `fpm.toml` file of project B to enable this? (I found a lot of info on how to include dependencies from github repos, but none for local projects)
Thanks!

**Comment by zoziha** [comment]
Hello,
You might be looking for this? ([local-dependencies](https://fpm.fortran-lang.org/en/spec/manifest.html#local-dependencies))

Currently the `path` is still relative to the top-level project's `fpm.toml`. Before the #719 merge, I thought `path` local dependencies would be a little less workable.

**Reply by HugoMVale** [reply]
Thanks, that is exactly what I was looking for.
